### ISSUE TITLE
Problematic use of receivers when having a primary ISAN Quad setup

### ISSUE THREAD

**Issue by abiosbug** [fixerid/sb-projects#7]
I've been trying to add your module to my Jilted Deluxe version I'm designing. - https://github.com/abiosbug/Jilted  TL;DR; I want to use your system with at least 2 other systems: compass & waypoints.
After extensive debugging. I noticed each of your ISAN scripts is setting the default ISAN quad receivers too. since it's still setting `:bt` `:ct` and `:dt`.  I'm still pretty new to this game, but it seems it doesn't cope well with multiple scripts setting and reading the same device at the very same yolol tick.  Maybe a unique `ChipWait` value for each chip might help with this, so the running loops don't access those 3 receivers at the same tick, it's still undesirable to do this imho.
Currently I'm testing by removing as much quad code as possible without breaking ISAN. I'll keep you posted.

**Comment by abiosbug** [comment]
The thing is, as soon as you address a global variable that does NOT exist in a device or a memory chip, the whole line will be skipped.   
So looking at your ISAN scripts, you can actually strike out each line that addresses `:b :bt :c :ct d: dt` in yolol as those variables are not supposed to exist without a Quad setup.  The ISAN code depends on those lines failing to determine if it should run it's mono loop or quad loop.  
I'm no expert, but Firestar is, he managed to get ISAN for 3 mono receivers at the same time with [just 10 lines of yolol](https://gitlab.com/Firestar99/yolol/-/jobs/1582394315/artifacts/file/build/yolol/compass/isan3in.yolol).  He even added a transformation matrix set along with his awesome compass module.  I'm currently testing if I can't just use his rotational data to have your nav-only script work much more efficient as his ISAN feedback loop seems faster and more compatible since it doesn't interfere with the default Quad setup.  
I'll keep you posted 🤞

**Comment by abiosbug** [comment]
Trying the following translation table atm, hope it just works:  
|      | NAVCAS | Compass | Matrix |
| :----  | ------ | ------- | ------ |
| FRONT  | a & at | kf & kt |        |
| F X    | xa     | fx      | jx     |
| F Y    | ya     | fy      | jy     |
| F Z    | za     | fz      | jz     |
| BACK   | j & jt | ke & kt |        |
| B X    | xj     | ex      | ix     |
| B Y    | yj     | ey      | iy     |
| B Z    | zj     | ez      | iz     |
| DOWN   | a & at | kf & kt |        |
| F X    | xk     | gx      | kx     |
| F Y    | yk     | gy      | ky     |
| F Z    | zk     | gz      | kz     |

**Comment by abiosbug** [comment]
I can confirm this actually works.  I'm not 100% if the testmode 's switched axis would affect your script, it somehow doesn't affect the compass script. 
TL;DR; by using the translation table above you can dump the 3 isan chips and reuse the values from compass (or use the compass-isan3in1 instead).
